packages/spotlight/src/spotlight.js

```diff
@@ -571,14 +596,15 @@ const Spotlight = (function () {
 		 * @public
 		 */
 		add: addContainer,
-
 		unmount: function (containerId) {
 			if (!containerId || typeof containerId !== 'string') {
+				// are we checking for all the missing params? looks little random to check just for this.
 				throw new Error('Please assign the "containerId"!');
 			}
 			unmountContainer(containerId);
 		},
 
+		// how is this differnet from unmount?
 		/**
```

Comment:
`Spotlight.remove(containerId)` actually removes the specified `containerId` from the internal collection of containers and is not available to be used again (unless explicitly re-added). `Spotlight.unmount(containerId)` retains the `containerId` in the collection, but de-activates the container. This is used when unmounting a container that you anticipate needing to re-mount again later (`Panel` for instance).